packages/twentytwenty-theme/src/components/post/post.js

```diff
@@ -114,7 +114,6 @@ const FeaturedImage = styled(FeaturedMedia)`
 
   > div {
     position: relative;
-    width: 100vw !important;
```

Comment:
This was causing overflow on the Post page and I could not see a good reason why it was there. Am I wrong @segunadebayo 🙂?

Reply:
> This was causing overflow on the Post page and I could not see a good reason why it was there. Am I wrong @segunadebayo 🙂?

Could you get back to me on this when you get a chance @segunadebayo ? 🙂

Reply:
Hey Michal. You're right, there's no reason to have it there. It was a carryover code from the mars-theme. Thanks for catching that.

Sorry for my late reply. I'm available today 😀

Reply:
Thanks Segun! :) No worries, I know that you were on holiday yesterday! 🙂